examples/xfixes-cursor-notify.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/python3
+#
+# examples/xfixes-cursor-notify.py -- demonstrate the XFIXES extension
+# CursorNotify event.
+#
+#    Copyright (C) 2022
+#      Dan Isla <dan.isla@gmail.com>
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public License
+# as published by the Free Software Foundation; either version 2.1
+# of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the
+#    Free Software Foundation, Inc.,
+#    59 Temple Place,
+#    Suite 330,
+#    Boston, MA 02111-1307 USA
+
+# Python 2/3 compatibility.
+from __future__ import print_function
+
+import sys
+from Xlib.display import Display
+from Xlib.ext import xfixes
+
+def main():
+    display = Display()
+
+    if not display.has_extension('XFIXES'):
+        if display.query_extension('XFIXES') is None:
+            print('XFIXES extension not supported', file=sys.stderr)
+            return 1
+
+    xfixes_version = display.xfixes_query_version()
+    print('Found XFIXES version %s.%s' % (
```

Comment:
It would be nice to use `.format(...)` here instead of string interpolation to be aligned with the rest files.

Reply:
Fixed. Thanks.